etcdserver/api/rafthttp/stream.go

```diff
@@ -214,6 +216,14 @@ func (cw *streamWriter) run() {
 					batched++
 				}
 
+				// snapshot sends are tracked via separate metrics https://github.com/etcd-io/etcd/pull/9997
+				// heartbeats are tracked via prober https://github.com/etcd-io/etcd/pull/10022
+				// TODO: track other messages?
+				if m.Type == raftpb.MsgProp ||
```

Comment:
this is not the sending time (flushing the buffer to the network card). we actually send the messages at line 211 when flush is called explicitly.

Reply:
i would rather just remove this metrics. it is not easy to measure after all with buffers all over the places.

Reply:
@xiang90 Good point. For now, tracking snapshots and heartbeats is good enough. Will come back to this later.